src/js/api/build.js

```diff
@@ -296,6 +296,84 @@ ripe.Ripe.prototype.getBuildArtifactsP = function(name, options) {
     });
 };
 
+/**
+ * Retrieves the build artifacts by name and version and for
+ * the requested branch.
+ *
+ * @param {String} name The name of the build artifacts.
+ * @param {Object} options An object with options, such as:
+ *  - 'branch' - The branch of the build.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getBuildLocalArtifacts = function(name, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const branch = options.branch === undefined ? null : options.branch;
+    const url = `${this.url}builds_local/${name}/artifacts`;
+    const params = {};
+    if (branch !== undefined && branch !== null) {
+        params.branch = branch;
+    }
+    options = Object.assign(options, {
+        url: url,
+        method: "GET",
+        auth: true,
+        params: params
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+/**
+ * Retrieves the build artifacts by brand name and version and for
+ * the requested branch.
+ *
+ * @param {String} name The name of the brand of the build artifacts.
+ * @param {Object} options An object with options, such as:
+ *  - 'branch' - The branch of the build.
+ * @returns {Promise} The build artifacts (as a promise).
+ */
+ripe.Ripe.prototype.getBuildLocalArtifactsP = function(name, options) {
+    return new Promise((resolve, reject) => {
+        this.getBuildLocalArtifacts(name, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request));
+        });
+    });
+};
+
+/**
+ * Retrieves all the artifacts of all the local builds.
+ *
+ * @param {Object} options An object of options to configure the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getLocalArtifacts = function(options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = `${this.url}builds/local_artifacts`;
+    options = Object.assign(options, {
+        url: url,
+        method: "GET",
+        auth: true
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+/**
+ * Retrieves all the artifacts of all the local builds.
+ *
+ * @param {Object} options An object of options to configure the request.
+ * @returns {Promise} The builds' artifacts (as a promise).
+ */
+ripe.Ripe.prototype.getLocalArtifactsP = function(options) {

```

Comment:
Change the order to reflect what's in RIPE Core. `getBuildLocalArtifacts` accesses nested resources (the artifacts from the build) so it should come first.